public/styles/component.scss

```diff
@@ -120,20 +120,25 @@ kbn-dis doc-table .kbnDocViewer__warning {
     flex-grow: 0;
 }
 
-.osButtonsStyle {
-    display: grid;
-    grid-template-columns: repeat(5, 1fr);
-    grid-gap: 10px;
+.message {
```

Comment:
**nitpick**: this class name is generic and could have conflicts with the styles of other plugins, we should use scoped names, for example prefixing it with `wz`, `wazuh`.

Reply:
Fixed in this [PR](https://github.com/wazuh/wazuh-kibana-app/pull/4837)